app/views/overview/_service-instance-row.html

```diff
@@ -97,59 +113,91 @@
           </div>
         </div>
         -->
-        <div class="row">
-          <div class="col-sm-12" ng-if="row.description">
-            <p class="pre-wrap" ng-bind-html="row.description | linky"></p>
+        <div ng-switch="row.instanceStatus">
+          <div ng-switch-when="deleted" class="row">
+            <div class="col-sm-12">
+              <div class="alert word-break alert-warning">
+                <span class="pficon pficon-warning-triangle-o" aria-hidden="true"></span>
+                <span class="sr-only">warning</span>
+                <span class="strong">The service was marked for deletion</span>
+                <span class="strong" am-time-ago="row.apiObject.metadata.deletionTimestamp"></span>
+              </div>
+            </div>
           </div>
-        </div>
-        <div class="section-title" ng-if="row.isBindable || row.bindings">
-          Bindings
-        </div>
-        <div class="row overview-bindings" ng-repeat="(name, binding) in row.bindings">
-          <div class="col-sm-5 col-md-6">
-            <div ng-if="!(row.state.applicationsByBinding[binding.metadata.name] | size)">
-              <h3>
-                <div class="component-label">
-                  Secret
+          <div ng-switch-when="failed" class="row">
+            <div class="col-sm-12">
+              <div class="alert word-break alert-danger">
+                <span class="pficon pficon-error-circle-o" aria-hidden="true"></span>
+                <span class="sr-only">error</span>
+                <span class="strong">{{row.instanceError.message}}</span>
+              </div>
+            </div>
+          </div>
+          <div ng-switch-when="pending" class="row">
+            <div class="col-sm-12">
+              <div class="alert word-break alert-info">
+                <span class="pficon pficon-info" aria-hidden="true"></span>
+                <span class="sr-only">info</span>
+                <span>{{row.pendingMessage}}</span>
```

Comment:
I think a message should always be there, but we might want a fallback just in case.

Could be just

```html
<span class="strong">The service is not yet ready.</span>
<span ng-if="row.pendingMessage">{{row.pendingMessage}}</span>
```